08_API_projects/074_github_finder_part_2_fetching_data_git_file.js

```diff
@@ -0,0 +1,17 @@
+class Github {
+    constructor() {
+        this.client_id = '2dd5b9a1915d53578ace';
+        this.client_secret = 'fa8b5124118056a822c9191aba7d747b1f2e8f7d';
```

Comment:
is this your client secret? or some generic one? if it is then please remove the client secret when checking in as your repo is public.